.github/workflows/ci.yml

```diff
@@ -18,8 +18,11 @@ permissions:
 jobs:
   # Avoid `push` and `pull_request` running the same job twice
   check_if_skip_duplicate_job:
-    runs-on: [self-hosted, linux, nix]
+    # runs-on: [self-hosted, linux, nix]
+    runs-on: ubuntu-latest
     steps:
+      - uses: DeterminateSystems/nix-installer-action@main
+      - uses: DeterminateSystems/magic-nix-cache-action@main
       - id: skip_check
         uses: fkirc/skip-duplicate-actions@v5
         with:
```

Comment:
What happened here? Somehow this also took my CI commit x) I think github merge got confused with the merges

Reply:
This is extremely strange. I never merge other branches than master in my local branch.

Reply:
I've read before that git merge doesn't deal super well with reverts

Reply:
Ah no, reverting *merges* is tricky, but reverting individual commit should work :thinking:

Reply:
Oh, I'm the one who reverted a merge x) All is explained.